modules/@ergonode/authentication/src/store/authentication/actions.js

```diff
@@ -60,7 +60,13 @@ export default {
                     refreshToken,
                 });
 
-                await dispatch('getUser');
+                const {
+                    language,
+                } = await dispatch('getUser');
+
+                if (language) {
+                    this.$setInterfaceLanguage(language);
```

Comment:
What if user doesn't have language? Should not we have default language? Maybe language = SOME_DEFAULT_LANGUAGE?

Reply:
This is solved in the i18n plugin